test/views/components/QuestionSpec.scala

```diff
@@ -51,9 +51,15 @@ class QuestionSpec extends WordSpec with Matchers with GuiceOneAppPerSuite {
     val a1 = Answer(Text(ans1), Some(Text(ans1Hint)), "/yes")
     val a2 = Answer(Text(ans2), Some(Text(ans2Hint)), "/no")
     val a3 = Answer(Text(ans3), Some(Text(ans3Hint)), "/dontknow")
+    val leading = Text("You can buy", "Gwallwch brynu")
+    val bp1 = Text("apples", "afalau")
+    val bp2 = Text("oranges", "orennau")
+    val bp3 = Text("pears", "gellyg")
+    val bpList: BulletPointList = BulletPointList(leading, Seq(bp1, bp2, bp3))
+
     val answers = Seq(a1, a2, a3)
     val horizontalAnswers = Seq(a1.copy(hint = None), a2.copy(hint = None))
-    val question = Question(Text(q1), Seq(para1), answers)
+    val question = Question(Text(q1), Seq(bpList,para1), answers)
```

Comment:
As you have added a bullet point list to the components in the question it would be good to check the bullet point list is in the html document.

Reply:
Fixed